BLUESPAWN-client/src/hunt/hunts/HuntT1131.cpp

```diff
@@ -16,47 +17,74 @@ namespace Hunts {
 		dwTacticsUsed = (DWORD) Tactic::Persistence;
 	}
 
-	int HuntT1131::EvaluatePackages(Registry::RegistryKey key, std::vector<std::wstring> vPackages, std::wstring name, Reaction reaction) {
+	int HuntT1131::ScanCursory(const Scope& scope, Reaction reaction) {
+		LOG_INFO(L"Hunting for " << name << L" at level Cursory");
+		reaction.BeginHunt(GET_INFO());
+
 		int detections = 0;
 
-		FileSystem::File* file;
+		// LSA Configuration
+		auto lsa = RegistryKey{ HKEY_LOCAL_MACHINE, L"SYSTEM\\CurrentControlSet\\Control\\Lsa" };
 
-		for (auto package : vPackages) {
-			if (package == L"\"\"") {
-				continue;
+		for (auto PackageGroup : { L"Authentication Packages", L"Notification Packages" }) {
+			auto Packages = lsa.GetValue<std::vector<std::wstring>>(PackageGroup);
+			if (Packages) {
+				for (auto Package : Packages.value()) {
+					if (Package != L"\"\"") {
```

Comment:
Checking if package is empty quotes; next statement assumes there are no quotes - is this valid?

Reply:
yes